src/main/java/net/sf/jabref/gui/importer/fetcher/ISBNtoBibTeXFetcher.java

```diff
@@ -101,7 +102,8 @@ public boolean processQuery(String query, ImportInspector inspector, OutputPrint
                 bibtexString = scan.useDelimiter("\\A").next();
             }
 
-            BibEntry entry = BibtexParser.singleFromString(bibtexString);
```

Comment:
I'm not so sure that it is always a good idea to use `ImportFormatPreferences.fromPreferences` for the fetchers. For example the encoding should be specified by the fetcher and not by the user preferences.